courses/models.py

```
@@ -38,25 +48,40 @@
+class CourseTeacher(BaseModel):
     ROLE_CHOICES = [
         ('teacher', 'Teacher'),
         ('assistant', 'Teaching Assistant'),
     ]
-    classroom = models.ForeignKey(Classroom, on_delete=models.CASCADE, related_name='teachers')
+    course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name='teachers')
     teacher = models.ForeignKey(
         User,
         on_delete=models.CASCADE,
-        limit_choices_to={'role': 'teacher'},
         related_name='teaching_roles'
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)